plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -351,7 +351,12 @@ class GeneralServiceFormSection extends Component {
               </FormGroupHeading>
             </FieldLabel>
             <FieldAutofocus>
-              <FieldInput name="id" type="text" value={data.id} />
+              <FieldInput
+                name="id"
+                type="text"
+                value={data.id}
+                disabled={/\/edit\//g.test(global.location.hash)}
```

Comment:
I know this might be very nitty picky, but would you mind moving this in a variable that gives a bit more context? Something like 
```js 
const isEditPage = /\/edit\//g.test(global.location.hash);
```
maybe?

/cc @juliangieseke I know you have strong opinions against too many variables, would you still agree on this one or am I a prisoner of my own customs?

Reply:
Well, putting this into a variable because it is hard to follow the inline regex would be a fix for a problem which should be fixed in a different way - IMHO a simple `global.location.hash.includes('edit')` would be sufficient and easier to read/understand 🤔 (assuming babel will polyfill it)